cves/CVE-2017-5638.yml

```diff
@@ -45,7 +45,12 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  Apache Struts uses the Jakarta Multiparser, and it generates an exception when parsing 
```

Comment:
What is the "Jakarta Multiparser" and what does it do? A brief explanation might help those of us not familiar with Apache Struts.

Reply:
I updated the comment to be more descriptive, Jakarta is basically a built in parser that is part of Struts and is used to parse input from HTTP POST requests.